01_prepare_host.sh

```diff
@@ -12,13 +12,29 @@ if [[ $(id -u) == 0 ]]; then
 fi
 
 if [[ $OS == ubuntu ]]; then
+  sudo apt-get update
   sudo apt -y install python3-pip
+
+  # Set update-alternatives to python3
```

Comment:
There seems to be duplication, do we still need to keep the same configuration here?: https://github.com/metal3-io/metal3-dev-env/blob/deb9086f7e42c49a45265e58bc21e68b6f9f83be/vm-setup/roles/packages_installation/tasks/ubuntu_required_packages.yml#L42

Reply:
Good catch, thanks. Fixed. I didn't remove the same thing from centos_install*.sh, because that will be done when moving centos_install* script to Ansible.